cw_monero/lib/api/wallet.dart

```diff
@@ -35,16 +35,13 @@ String getSeed() {
   if (cakepolyseed != "") {
     return cakepolyseed;
   }
-  final polyseed = monero.Wallet_getPolyseed(wptr!, passphrase: '');
-  if (polyseed != "") {
-    return polyseed;
-  }
```

Comment:
why was this removed?
if `Wallet_getPolyseed` function would eventually try to `Wallet_getCacheAttribute` since Polyseed is saved in cache, then yeah that makes sense, and we can later remove `Wallet_getPolyseed` since it's useless

Reply:
It is the "soon(tm)" to be available in upstream monero code function for polyseed, we don't use it so it would always report empty string - there is no harm in keeping that though